src/SourceBuild/content/repo-projects/test-templates.proj

```diff
@@ -6,7 +6,7 @@
 
   <ItemGroup>
     <RepositoryReference Include="arcade" />
-    <RepositoryReference Include="templating" />
+    <RepositoryReference Include="source-build-reference-packages" Condition="'$(DotNetBuildSourceOnly)' == 'true'" />
```

Comment:
How did you determine that list? Does this represent the source-build or the vertical build graph? As an example, is the removal of templating here due to the repository not being a dependency for source build but for vertical build? cc @mmitche

Reply:
The removal of `templating` here is because there's literally no dependency on the templating repo output.

But you bring up a good point about the difference between vertical and source build. In some cases, there are dependencies that exist purely to satisfy source build. In order words, the dependencies for vertical build would be a subset of the dependencies for source build. I'll investigate that more so that we can define these depends with the source-only flag as appropriate.

Reply:
Yeah we will need to go and annotate these dependencies with SB properties in some cases. The "Microsoft" graph should emulate as closely the active flow that happens via Maestro on a per-release basis. The SB graph will start with this flow, and then add new edges where we repos are redistributing implementation packages of older versions.